chcemvediet/apps/anonymization/models.py

```diff
@@ -100,14 +99,105 @@ def save(self, *args, **kwargs):
                 self.file.name = random_string(10)
                 self.size = self.file.size
                 self.name = adjust_extension(self.attachment.name, self.content_type)
-
         super(AttachmentNormalization, self).save(*args, **kwargs)
 
     def __unicode__(self):
         return format(self.pk)
 
+class AttachmentRecognitionQuerySet(QuerySet):
+    def order_by_pk(self):
+        return self.order_by(u'pk')
+
+class AttachmentRecognition(FormatMixin, models.Model):
+
+    # May NOT be NULL
+    attachment = models.ForeignKey(Attachment)
+
+    # May NOT be NULL
+    successful = models.BooleanField(default=False,
+            help_text=squeeze(u"""
+                True if recognition has succeeded, False otherwise.
+                """))
+
+    # Filename may be empty; Random local filename is generated in save() when creating a new object
+    file = models.FileField(upload_to=u'attachment_recognitions', max_length=255, blank=True,
+            help_text=squeeze(u"""
+                Empty filename if recognition failed or recognition didn't create any file.
+                """))
+
+    # May be empty: Extension automatically adjusted in save() when creating new object.
+    name = models.CharField(max_length=255, blank=True,
+            help_text=squeeze(u"""
+                Attachment file name, e.g. "document.pdf". Extension automatically adjusted when
+                creating a new object. Empty, if file.name is empty.
+                """))
+
+    # May be NULL
+    content_type = models.CharField(max_length=255, null=True,
+            help_text=squeeze(u"""
+                Attachment content type, e.g. "application/pdf". The value may be specified even if
+                recognition failed.
+                """))
+
+    # May NOT be NULL; Automatically computed in save() when creating a new object if undefined.
+    created = models.DateTimeField(blank=True,
+            help_text=squeeze(u"""
+                Date and time the attachment was recognized. Leave blank for current time.
+                """))
+
+    # May be NULL; Automatically computed in save() when creating a new object.
+    size = models.IntegerField(null=True, blank=True,
+            help_text=squeeze(u"""
+                Attachment file size in bytes. NULL if file is NULL. Automatically computed when
```

Comment:
Je to `Attachment file size` v zmysle velkost prilohy, alebo velkost `Attachment` ako instancie? Samozrejme ze sa jedna o velkost `self.file`. Je to asi trocha matuce. Ci by mozno nebolo vhodne prepisat na `AttachmentRecognition file size`. Podobne na inych miestach a aj v `AttachemntNormalization`.

Reply:
Napis "Attachment recognition file size in...". A analogicky aj "Attachemnt normalization file size...". Kedze je to vysvetlujuci text, nie kod, nemusime davat presne identifikatory.

Reply:
:ok: